versions/blockspacerace_versions.js

```diff
@@ -1,9 +1,9 @@
 const blockspaceraceVersions = Object.freeze({
   "app-latest-tag": "v0.12.0",
-  "app-latest-sha": "67c90ede03d2543ce9b02726dfe17cf5aad1f20a",
-  "core-latest-tag": "v1.14.0-tm-v0.34.23",
-  "core-latest-sha": "ab4ef71b202dd9d27c879a820fa06e127d4234fa",
+  "app-latest-sha": "1943735bdefc751ae98732760921d49a3534a70a",
+  "core-latest-tag": "v1.15.0-tm-v0.34.23",
+  "core-latest-sha": "77a1de71feadebaa89c5ecaef0309fd8b9f2f921",
```

Comment:
no one needs to worry about core since its committed to in both app and node via their hashes